src/components/SingleDatePickerInputController.jsx

```diff
@@ -109,6 +109,10 @@ const defaultProps = {
 
   // accessibility
   isFocused: false,
+  ariaLabel: undefined,
+  ariaInvalid: undefined,
+  ariaDescribedby: undefined,
+  screenReaderMessage: 'Date',
```

Comment:
Not from your code, but this is probably a bad value to set as a default. It's both set to something that's not particularly useful and also is hardcoded in English.